torch/csrc/jit/interpreter.cpp

```diff
@@ -1053,6 +1053,14 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
       if (i > 0) {
         out << "during call ";
```

Comment:
"during call" formatting here is going to cause the formatting to be different depending of if things have been inlined. I think we should have a test that runs the model with/without inlining and checks the stack trace appears byte-for-byte the same.